docs/source/javascripts/docsearch.js

```diff
@@ -3,7 +3,8 @@ import docsearch from 'docsearch.js';
 const search = docsearch({
   apiKey: '5718722ffb11e109821befd53a1d9fde',
   indexName: 'places',
-  inputSelector: '#docsearch'
+  inputSelector: '#docsearch',
+  debug: true
```

Comment:
but unsure we want to activate debug on the production, maybe its something you did not add on purpose?